### PR TITLE
FE for historical work page

### PR DESCRIPTION
**Pull request by helenb**
[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1370032957)

### Description of Changes Made

- Styling for old work pages
- Note if @engineervix  renames the template / model some of this may need refactoring with his branch
- Tweaks a couple of things in the blog page
- Adds styling for embed blocks

### How to Test

View a work page in the local build: http://localhost:8000/work/voyage-of-discovery-nasa-jpl/
Or in the pattern library: http://localhost:8000/pattern-library/pattern/patterns/pages/work/work_detail.html

### Screenshots

<details>
  <summary>Expand to see more</summary>

Screenshot 2024-01-23 at 13 14 34

Screenshot 2024-01-23 at 13 14 21

</details>

### MR Checklist

- [x] Add a description of your pull request and instructions for the reviewer to verify your work.
- [x] If your pull request is for a specific ticket, link to it in the description.
- [x] Stay on point and keep it small so the merge request can be easily reviewed.
- [x] Tests and linting passes.

#### Unit tests

- [ ] Added
- [x] Not required

#### Documentation

- [ ] Updated build docs
- [ ] Updated editor guidelines (https://docs.google.com/document/d/1PAWccdQ4tfaZsrEWmpDhvP3GH5RRmBOARFVp4b-kje8/edit?usp=sharing)
- [x] Not required

#### Browser testing

- [x] I have tested in the following browsers and environments (edit the list as required)
  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android
- [ ] Not required

#### Data protection

- [x] Not relevant
- [ ] This adds new sources of PII and documents it and modifies Birdbath processors accordingly

#### Accessibility

- [x] Automated WCAG 2.1 tests pass
- [x] Manual WCAG 2.1 tests completed
- [x] I have tested in a screen reader
- [x] I have tested in high-contrast mode
- [ ] Any animations removed for prefers-reduced-motion
- [ ] Not required

#### Sustainability

- [x] Images are optimised and lazy-loading used where appropriate
- [ ] SVGs have been optimised
- [ ] Perfomance and transfer of data considered
- [ ] If JavaScript is needed alternatives have been considered
- [x] New font variants have not been added
- [ ] Not required

#### Pattern library

- [x] The pattern library component for this template displays correctly, and does not break parent templates
- [ ] The styleguide is updated if relevant
- [ ] Changes are not relevant the pattern library

### CODE REVIEW

File: tbx/project_styleguide/templates/patterns/atoms/avatar/avatar.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
     <picture>
         <source media="(max-width: 598px)" srcset="{{  avatar_mobile.url }} 1x, {{ avatar_desktop.url }} 2x" />
         <source media="(min-width: 599px)" srcset="{{ avatar_desktop.url }} 1x, {{ avatar_desktop_retina.url }} 2x" />
-        <img src="{{ avatar_desktop.url }}" alt="{{ avatar_desktop.alt }}" {% if lazy_load %}loading="lazy"{% endif %} class="avatar__image" />
+        <img src="{{ avatar_desktop.url }}" alt="" {% if lazy_load %}loading="lazy"{% endif %} class="avatar__image" />
```
Comment (helenb) [inline tbx/project_styleguide/templates/patterns/atoms/avatar/avatar.html:10]: The example I tested had a terrible alt text for the author image - changed to empty and made more explicit in the author template instead.

File: tbx/project_styleguide/templates/patterns/pages/blog/blog_detail.html [hunk 2]
```diff
@@ -26,17 +26,17 @@
...
         <div class="authors">
             {% for page_author in page.authors.all %}
-                {% include "patterns/molecules/author/author.html" with author=page_author.author item=page %}
+                {% include "patterns/molecules/author/author.html" with author=page_author.author %}
             {% endfor %}
         </div>
 
         <div class="streamfield__title">
-            <h1 class="heading heading--one">{{ page.title }}</h1>
+            <h1 class="heading heading--one-b">{{ page.title }}</h1>
```
Comment (helenb) [inline tbx/project_styleguide/templates/patterns/pages/blog/blog_detail.html:34]: Tweak to blog heading size to match instructions in design docs

File: tbx/project_styleguide/templates/patterns/pages/work/work_detail.html [hunk 3]
```diff
@@ -17,29 +17,75 @@
...
+                <span class="sr-only">Related work categories</span>
+                {% for tag in page.related_services.all %}
+                    {% include "patterns/atoms/tag/tag.html" with tag_slug=tag.slug tag_name=tag.name tag_link_base=work_index_url %}
+                {% endfor %}
+            </div>
+        </div>
 
-        {% include "patterns/molecules/author/author.html" with item=page author=author tag_link_base=work_index_url %}
+        {% for intro_block in page.body %}
+            {% if intro_block.block_type == 'intro' %}
```
Comment (helenb) [inline tbx/project_styleguide/templates/patterns/pages/work/work_detail.html:47]: Bit of a hack to make the streamfield intro block come at the right position for the design at the top of the work page.

File: tbx/project_styleguide/templates/patterns/atoms/avatar/avatar.html (hunk 1 above)
Comment (chris-lawton) [reply tbx/project_styleguide/templates/patterns/atoms/avatar/avatar.html:10]: Checking we're ok that all avatars will be decorative?

File: tbx/project_styleguide/templates/patterns/pages/work/work_detail.html [hunk 4]
```diff
@@ -17,29 +17,75 @@
...
-        {% include_block page.call_to_action %}
+        <div class="work-meta mt-spacerSmall mb-spacerSmall lg:mb-spacerLarge">
+            {% if page.client %}
+                <p class="work-meta__client">
+                    {{ page.client }}
+                </p>
+            {% endif %}
+
+            <p class="work-meta__meta">
+                <time datetime="{{ item.date|date:'Y-m-d' }}">{{ page.date }}</time>
```
Comment (chris-lawton) [inline tbx/project_styleguide/templates/patterns/pages/work/work_detail.html:60]: I wonder if we should set the date format globally to `|date:'Y-m-d'` so we don't have to add the filter everywhere it's used.

**Comment by chris-lawton** [comment]
Have we considered what will happen when a page doesn't have a listing image? At the moment we don't check `listing--image.html` for it so there's a gap where the image tag has rendered but no img src:

Screenshot 2024-01-23 at 16 56 21

Might be best to stick on the safe side and add a conditional in rather than rely on editors to add a listing image.

**Review by chris-lawton — APPROVED** [review]
Looks great @helenb - thanks.

File: tbx/project_styleguide/templates/patterns/atoms/avatar/avatar.html (hunk 1 above)
Comment (helenb) [reply tbx/project_styleguide/templates/patterns/atoms/avatar/avatar.html:10]: I've done a quick check of everywhere we use it so far, and there is always a name alongside it. I did spot an issue with the styleguide yaml which I've updated in 3c181f3002cba68c41414e9b2ad20dc92cf6d812

File: tbx/project_styleguide/templates/patterns/pages/work/work_detail.html (hunk 4 above)
Comment (helenb) [reply tbx/project_styleguide/templates/patterns/pages/work/work_detail.html:60]: good idea I'll check how to do that

File: tbx/project_styleguide/templates/patterns/pages/work/work_detail.html [hunk 5]
```diff
@@ -17,29 +17,75 @@
...
+            {% endif %}
+        {% endfor %}
 
         {% if page.related_works %}
-            {% include "patterns/atoms/section-title/section-title.html" with title="More of our work" %}
-            {% include "patterns/organisms/work-listing/work-listing.html" with works=page.related_works %}
-            {% include "patterns/atoms/see-more/see-more.html" with link=work_index_url text="See more case studies" %}
+            {% include "patterns/atoms/section-title/section-title.html" with title="More" aria_label="Related work pages" classes="section-title--related-posts" %}
+            {% comment %} {% include "patterns/organisms/work-listing/work-listing.html" with works=page.related_works %}
+            {% include "patterns/atoms/see-more/see-more.html" with link=work_index_url text="See more case studies" %} {% endcomment %}
```
[the remark this answers is not in the document]
Comment (helenb) [reply tbx/project_styleguide/templates/patterns/pages/work/work_detail.html:78]: whoops that was just old code I'd kept in for reference.

[the remark this answers is not in the document]
Comment (helenb) [reply tbx/project_styleguide/templates/patterns/pages/work/work_detail.html]: Removed in a123f0ac860ed4e8a20ddd17b01795f8d7aa4c30

**Comment by helenb** [comment]
> Have we considered what will happen when a page doesn't have a listing image? At the moment we don't check `listing--image.html` for it so there's a gap where the image tag has rendered but no img src:
> 
> Screenshot 2024-01-23 at 16 56 21
> 
> Might be best to stick on the safe side and add a conditional in rather than rely on editors to add a listing image.

Hmm I just checked that page and it does have images for all the related work pages - I will quickly check and see if there are any that don't.

**Comment by helenb** [comment]
> Hmm I just checked that page and it does have images for all the related work pages - I will quickly check and see if there are any that don't.

I've checked and all work pages have a feed image so don't think this is a big concern

File: tbx/project_styleguide/templates/patterns/pages/work/work_detail.html (hunk 4 above)
Comment (helenb) [reply tbx/project_styleguide/templates/patterns/pages/work/work_detail.html:60]: I'll add this to a list as for now I'm not sure and I'd like to get this merged.

**Pull request merged**